### PR TITLE
Basic control over system settings for superusers

### PR DESCRIPTION
**Pull request by madchap**
Addresses #2135 

**Note: DefectDojo is now on Python3.5 and Django 2.2.x Please submit your pull requests to the 'dev' branch as the 'legacy-python2.7' branch is only for bug fixes. Any new features submitted to the legacy branch will be ignored and closed.**

When submitting a pull request, please make sure you have completed the following checklist:

- [x] Your code is flake8 compliant 
- [x] Your code is python 3.5 compliant (specific python >=3.6 syntax is currently not accepted)
- [ ] If this is a new feature and not a bug fix, you've included the proper documentation in the ReadTheDocs documentation folder. https://github.com/DefectDojo/Documentation/tree/master/docs or provide feature documentation in the PR.
- [ ] Model changes must include the necessary migrations in the dojo/db_migrations folder.
- [ ] Add applicable tests to the unit tests.
- [ ] Add the proper label to categorize your PR 

Current accepted labels for PRs:
- Import Scans (for new scanners/importers)
- enhancement
- feature
- bugfix
- maintenance (a.k.a chores)
- dependencies

### CODE REVIEW

**Comment by valentijnscholten** [comment]
nO NeWlINe aT eND oF FIlE ;-)
But I like it.

File: dojo/urls.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 from django.conf import settings
 from django.conf.urls import include, url
-from django.conf.urls.static import static
```
Comment (Maffooch) [inline dojo/urls.py:3]: This function is used in line 204 here. Was the deletion intentional?

Comment (madchap) [reply dojo/urls.py:3]: Hmm, no.

**Comment by madchap** [comment]
Resolved @Maffooch point, as well as conflicts. @Maffooch if you can give this a quick look, it would be great :) Thanks.

**Comment by Maffooch** [comment]
Looks great now! Something weird happened when fixing conflicts that's causing Flake8 to fail.

**Pull request merged**